pandas/tests/arrays/test_datetimelike.py

```diff
@@ -108,14 +108,20 @@ def test_take_fill(self):
         result = arr.take([-1, 1], allow_fill=True, fill_value=pd.NaT)
         assert result[0] is pd.NaT
 
-        with pytest.raises(ValueError):
-            arr.take([0, 1], allow_fill=True, fill_value=2)
+        value = 2
```

Comment:
can you split this out from test_take_fill as a separate test, say test_take_fill_raises and parametrize over value.